MicrosoftTeams.schema.json

```diff
@@ -331,6 +331,107 @@
       "items": {
         "type": "object",
         "additionalProperties": false,
+        "configuration": {
+          "team": {
```

Comment:
"team" is the scope identifier -- @asnow003 

eventually we will have 
"configuration": {
  "team": {},
  "groupchat": {},
 "personal":{}  
}